### PR TITLE
moved wrap_litserve_start to utils

### PR DESCRIPTION
**Pull request by ankitsharma07**
#195 move wrap_litserve_start to utils

i moved the function inside src > litserve > utils and then reimported them inside all the tests files.

### CODE REVIEW

**Comment by ankitsharma07** [comment]
@aniketmaurya hi, some checks are failing but i have moved the functions and its usages, can you check?

File: src/litserve/utils.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 import pickle
 import uuid
 from typing import Coroutine, Optional
+from contextlib import contextmanager
+from litserve.server import LitServer
```
Comment (aniketmaurya) [inline src/litserve/utils.py:20]:
```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from litserve.server import LitServer
```

File: src/litserve/utils.py [hunk 2]
```diff
@@ -69,6 +71,18 @@ async def azip(*async_iterables):
         yield tuple(results)
 
 
+@contextmanager
+def wrap_litserve_start(server: LitServer):
```
Comment (aniketmaurya) [inline src/litserve/utils.py:75]:
```suggestion
def wrap_litserve_start(server: "LitServer"):
```

**Review by aniketmaurya — COMMENTED** [review]
Looks great @ankitsharma07! Added a suggestion to fix the failing tests.

**Review by aniketmaurya — APPROVED** [review]

**Pull request merged**